### ISSUE TITLE
ScalaCheck master compilation fails with many errors

### ISSUE THREAD

**Issue by eallik** [rickynils/scalacheck#148]
I'm sorry if this is just a temporary failure and I shouldn't be reporting this as an issue, but it's currently effectively preventing me from making a simple contribution to the project.

```scala
~/code/scala/scalacheck$ sbt
[info] Loading project definition from /Users/erik/code/scala/scalacheck/project
[info] Set current project to scalacheck (in build file:/Users/erik/code/scala/scalacheck/)
> compile
[info] Compiling 13 Scala sources to /Users/erik/code/scala/scalacheck/target/scala-2.10/classes...
[info] Compiling 6 Scala sources to /Users/erik/code/scala/scalacheck/js/target/scala-2.11/classes...
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/Prop.scala:24: object scalajs is not a member of package scala
[error] @scala.scalajs.js.annotation.JSExportDescendentClasses
[error]        ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/Prop.scala:25: object scalajs is not a member of package scala
[error] @scala.scalajs.js.annotation.JSExportDescendentObjects
[error]        ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:12: not found: object sbt
[error] import sbt.testing._
[error]        ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:13: object scalajs is not a member of package org
[error] import org.scalajs.testinterface.TestUtils.{loadModule, newInstance}
[error]            ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:21: not found: type Runner
[error] ) extends Runner {
[error]           ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:33: not found: type TaskDef
[error]   def deserializeTask(task: String, deserializer: String => TaskDef) = {
[error]                                                             ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:48: not found: type Task
[error]   abstract class BaseTask(override val taskDef: TaskDef) extends Task {
[error]                                                                  ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:67: not found: type Task
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                                                                       ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:61: not found: type EventHandler
[error]     def execute(handler: EventHandler, loggers: Array[Logger],
[error]                          ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:61: not found: type Logger
[error]     def execute(handler: EventHandler, loggers: Array[Logger],
[error]                                                       ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:62: not found: type Task
[error]       continuation: Array[Task] => Unit
[error]                           ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:67: not found: type EventHandler
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                          ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:67: not found: type Logger
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                                                       ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:48: not found: type TaskDef
[error]   abstract class BaseTask(override val taskDef: TaskDef) extends Task {
[error]                                                 ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:66: not found: type TaskDef
[error]   def rootTask(td: TaskDef) = new BaseTask(td) {
[error]                    ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:81: not found: type Task
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                                                                       ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:81: not found: type EventHandler
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                          ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:81: not found: type Logger
[error]     def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
[error]                                                       ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:75: not found: type TaskDef
[error]   def checkPropTask(taskDef: TaskDef) = new BaseTask(taskDef) {
[error]                              ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:69: not found: type TaskDef
[error]         checkPropTask(new TaskDef(td.fullyQualifiedName, td.fingerprint,
[error]                           ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:43: not found: type TaskDef
[error]   def serializeTask(task: Task, serializer: TaskDef => String) =
[error]                                             ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:43: not found: type Task
[error]   def serializeTask(task: Task, serializer: TaskDef => String) =
[error]                           ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:46: not found: type Task
[error]   def tasks(taskDefs: Array[TaskDef]): Array[Task] = taskDefs.map(rootTask)
[error]                                              ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:46: not found: type TaskDef
[error]   def tasks(taskDefs: Array[TaskDef]): Array[Task] = taskDefs.map(rootTask)
[error]                             ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:46: type mismatch;
[error]  found   : taskDefs.type (with underlying type Array[<error>])
[error]  required: ?{def map: ?}
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method booleanArrayOps in object Predef of type (xs: Array[Boolean])scala.collection.mutable.ArrayOps[Boolean]
[error]  and method byteArrayOps in object Predef of type (xs: Array[Byte])scala.collection.mutable.ArrayOps[Byte]
[error]  are possible conversion functions from taskDefs.type to ?{def map: ?}
[error]   def tasks(taskDefs: Array[TaskDef]): Array[Task] = taskDefs.map(rootTask)
[error]                                                      ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:52: not found: type SubclassFingerprint
[error]       val fp = taskDef.fingerprint.asInstanceOf[SubclassFingerprint]
[error]                                                 ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:53: not found: value loadModule
[error]       val obj = if (fp.isModule) loadModule(taskDef.fullyQualifiedName,loader)
[error]                                  ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:54: not found: value newInstance
[error]                 else newInstance(taskDef.fullyQualifiedName, loader)(Seq())
[error]                      ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:136: not found: type Framework
[error] final class ScalaCheckFramework extends Framework {
[error]                                         ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:139: not found: type SubclassFingerprint
[error]     new SubclassFingerprint {
[error]         ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:147: not found: type Fingerprint
[error]   def fingerprints: Array[Fingerprint] = Array(
[error]                           ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:156: not found: type Runner
[error]   ): Runner = new ScalaCheckRunner(args, remoteArgs, loader) {
[error]      ^
[error] /Users/erik/code/scala/scalacheck/src/main/scala/org/scalacheck/ScalaCheckFramework.scala:177: not found: type Runner
[error]   ): Runner = new ScalaCheckRunner(args, remoteArgs, loader) {
[error]      ^
[error] 33 errors found
```

**Comment by eallik** [comment]
that was on OS X by the way.

**Comment by rickynils** [comment]
@eallik Is it working for you now that 784ef23 has been merged?

**Comment by eallik** [comment]
Yes, I was able to successfully `sbt compile` the project now at cbd06b3.
